### PR TITLE
Check for demux while moving data to nosync

### PR DESCRIPTION
**Pull request by senthil10**
If "configured" so, check if the run is demultiplexed rather than just evaluating time.

### CODE REVIEW

File: taca/storage/storage.py [hunk 1]
```diff
@@ -24,19 +24,34 @@ def cleanup_nas(seconds):
 
     :param int seconds: Days/hours converted as second to consider a run to be old
     """
+    couch_info = CONFIG.get('statusdb')
```
Comment (vezzi) [inline taca/storage/storage.py]: maybe worth checking if these fields are in the config in order to control the termination

Comment (vezzi) [inline taca/storage/storage.py]: forget me, you make the check in methods.

File: taca/storage/storage.py [hunk 2]
```diff
@@ -24,19 +24,34 @@ def cleanup_nas(seconds):
...
+                        if misc.run_is_demuxed(run, couch_info):
+                            logger.info('Moving run {} to nosync directory'.format(os.path.basename(run)))
+                            shutil.move(run, 'nosync')
+                        elif os.stat(rta_file).st_mtime < time.time() - seconds:
+                            logger.warn('Run {} is older than given time, but it is not demultiplexed yet'
+                                        .format(run))
+                            sbt = "Run not demultiplexed - {}".format(run)
+                            msg = ("Run '{}' in '{}' is older then given threshold, but seems like it is not "
+                                  "yet demultiplexed".format(os.path.join(data_dir, run), host_name))
+                            misc.send_mail(sbt, msg, mail_recipients)
```
Comment (vezzi) [inline taca/storage/storage.py]: cool

**Comment by vezzi** [comment]
Great @senthil10 with this we have solved the problem of monitoring FCs that are not syncing to preproc servers

**Comment by senthil10** [comment]
Tack @vezzi  👍

**Pull request merged**